fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/services/CreateResourceServiceImplTest.java

```diff
@@ -140,47 +162,70 @@
 
     private static final Collection<URI> DIGESTS = singleton(URI.create("urn:sha1:12345abced"));
 
+    private static final List<String> cleanup_list = new ArrayList<>() ;
```

Comment:
Minor: so we keep the code in line with basic Java conventions, please change to: `cleanupList`